ui/components/app/edit-gas-fee-popover/edit-gas-item/index.scss

```diff
@@ -0,0 +1,56 @@
+.edit-gas-item {
+  border-radius: 24px;
+  color: $ui-4;
+  cursor: pointer;
+  font-size: 12px;
+  margin: 12px 0;
+  padding: 4px 12px;
+  height: 32px;
+
+  &-selected {
```

Comment:
This seems to represent a state, should this be `&--selected`?

Reply:
Fixed